ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/RejectionReason.scala

```diff
@@ -73,4 +73,16 @@ object RejectionReason {
   final case class SubmitterCannotActViaParticipant(details: String) extends RejectionReason {
     override def description: String = "Submitter cannot act via participant: " + details
   }
+
+  /** The ledger time of the transaction submission violated one of the
+    *  following constraints on ledger time:
+    *  - The difference between ledger time and the record time
+    *    in the ledger state at which the transaction was sequenced needs
+    *    to stay within bounds defined by the ledger.
+    *  - The ledger time has to be greater than or equal to the ledger time
```

Comment:
This sentence uses `ledger time` twice.

Reply:
Hm, that's because it's talking about the relation of two different ledger times. I have added a qualifier to the first "ledger time" to make it clearer.

Reply:
The sentence now reads: _"The ledger time of the transaction must be greater than or equal to the ledger time of any contract used by the transaction."_